tests/rest/client/test_room_access_rules.py

```diff
@@ -721,10 +727,12 @@ def test_revoke_3pid_invite_direct(self):
         )
 
     def test_check_event_allowed(self):
-        """Tests that AccessRules.check_event_allowed behaves accordingly"""
-        access_rules = RoomAccessRules(
-            self.access_rules_config, ModuleApi(self.hs, self.hs.get_auth()),
-        )
+        """Tests that RoomAccessRules.check_event_allowed behaves accordingly.
+
+        It tests that:
+            * forbidden users cannot join restricted rooms.
+            * forbidden users can join unrestricted rooms if they have an invite.
```

Comment:
```suggestion
            * forbidden users can only join unrestricted rooms if they have an invite.
```

This seems more accurate to me.